contracts/satoshi-bridge/src/zcash_utils/contract_methods.rs

```diff
@@ -97,12 +99,30 @@ impl Contract {
         #[callback_unwrap] last_block_height: u32,
     ) -> U128 {
         let expiry_height = last_block_height + self.get_config().expiry_height_gap;
+
+        // For withdrawals with Orchard bundle, calculate the expected net amount after fees
+        let (expected_recipient, expected_amount) = if orchard_bundle.is_some() {
+            let withdraw_fee = self.internal_config().withdraw_bridge_fee.get_fee(amount.0);
+            // The orchard amount is the withdrawal amount minus the withdraw fee and gas fee
+            // max_gas_fee defaults to 10000 satoshis if not provided
+            let gas_fee = max_gas_fee.map(|g| g.0).unwrap_or(10000);
```

Comment:
10,000 is the wrong value, see the comment above. And we should create a separate function for this.